packages/components/inputs/select-utils/package.json

```diff
@@ -31,7 +31,7 @@
     "@emotion/styled": "^11.3.0",
     "lodash": "4.17.21",
     "prop-types": "15.7.2",
-    "react-select": "4.3.1"
+    "react-select": "^5.2.1"
```

Comment:
We need to update all other packages to v5 as well.

PS: we can maybe extract the upgrade to a separate PR.